tests/e2e_test.go

```diff
@@ -133,15 +133,22 @@ var _ = ginkgo.BeforeSuite(func() {
 		teleporterInfo.DeployTeleporterRegistry(subnet, fundedKey)
 	}
 
+	// Default balance for funded accounts
+	fundedBalance := units.Avax * 100
+	// Underfunded balance for validators
+	underfundedBalance := 2048*units.NanoAvax - 1
+
 	// Convert the subnets to sovereign L1s
 	for _, subnet := range localNetworkInstance.GetL1Infos() {
 		localNetworkInstance.ConvertSubnet(
 			networkStartCtx,
 			subnet,
 			teleporterTestUtils.PoAValidatorManager,
-			[]uint64{units.Schmeckle, units.Schmeckle},
+			[]uint64{units.Schmeckle, units.Schmeckle, units.Schmeckle, units.Schmeckle},
+			[]uint64{fundedBalance, fundedBalance, fundedBalance, underfundedBalance},
```

Comment:
This does test it but very implicitly. I would document the reason for why we are adding a validator and perhaps add an explicit test that using signature aggregator that explicitly confirms that underfunded node is not a member of the aggregate signature.